dojo-dungeon/src/dojo/rooms/Room.java

```diff
@@ -140,4 +144,23 @@ public void closeDoors() {
   public Tile getStartTile() {
     return levelRoom.level().startTile();
   }
+
+  /**
+   * Add a description dialog to this room, that is shown each time the player enters the room.
+   *
+   * @param description the description of the room
+   * @param title the title of the room
```

Comment:
Find es irgendwie seltsam das ich Titel und Beschreibung in einem übergebe/speicher.
für mich hätte ein Room zwei Attribute `String title`, `description`